pnpm-lock.yaml

```diff
@@ -8,13 +8,18 @@ overrides:
   socket.io: ^4.7.0
   rollup: ^4.2.0
 
+patchedDependencies:
+  ember-cli-blueprint-test-helpers@0.19.2:
+    hash: ss3hfjhjw5zkh5y42gamk2lhzy
+    path: patches/ember-cli-blueprint-test-helpers@0.19.2.patch
+
 importers:
 
   .:
     dependencies:
       '@babel/core':
         specifier: ^7.24.4
-        version: 7.24.4
+        version: 7.24.4(supports-color@8.1.1)
```

Comment:
There are a lot of lock changes, but I used the pnpm version that was listed in the package.json (8.10). I guess it's possible someone else used a different version in the past?